pol-core/bscript/compiler/ast/DefaultConstructorFunction.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "bscript/compiler/ast/UserFunction.h"
+
+namespace Pol::Bscript::Compiler
+{
+class DefaultConstructorFunction : public UserFunction
+{
+public:
+  DefaultConstructorFunction( const SourceLocation&, const std::string& class_name );
```

Comment:
not 100% sure this will stay, but currently there needs to be _some_ AST `Function` called when doing `FunctionCall()` ... creating a meta-function seems like the easiest way... similar to how we create `_x_iter` for loops